src/Umbraco.Web/WebApi/Filters/FilterAllowedOutgoingMediaAttribute.cs

```diff
@@ -77,9 +78,20 @@ public override void OnActionExecuted(HttpActionExecutedContext actionExecutedCo
 
         protected virtual void FilterItems(IUser user, IList items)
         {
-            FilterBasedOnStartNode(items, user);
+
+            Guid? dataTypeId = Guid.TryParse(HttpContext.Current.Request.QueryString.Get(TreeQueryStringParameters.DataTypeId), out var temp) ? (Guid?)temp : null;
```

Comment:
There should be no changes to this file at all, there's no need since no changes go through the media controller (this is also dangerous to do)